examples/features/gracefulstop/README.md

```diff
@@ -0,0 +1,31 @@
+# Graceful Stop
+
+This example demonstrates how to gracefully stop a gRPC server using
+Server.GracefulStop().
```

Comment:
Backticks?

Reply:
```suggestion
`Server.GracefulStop()`.
```

Reply:
GitHub has a somewhat hidden feature that lets you offer suggestions by editing the text selected in the comment box . Then the PR author can click "accept" instead of checking out the repo and editing it.

Reply:
https://haacked.com/archive/2019/06/03/suggested-changes/

Reply:
Added backticks. Thanks